docs/deployments/deploy-chrome-extension.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Deploy a Chrome Extension to production
+description: Learn how to deploy a Chrome Extension to production with Clerk.
+---
+
+> [!WARNING]
+> This guide assumes you are using the Plasmo framework. If you need help applying these concepts to another framework, [contact support](https://clerk.com/contact/support).
+
+## Create a production instance
+
+For Clerk production instances, there must be a domain associated with the instance. Even though there may not be a web application associated with your Chrome Extension, a domain is still required. Follow the [guide on deploying your Clerk app to production](/docs/deployments/overview).
+
+## Update your `.env.production` file
+
+<SignedIn>
+  Add your publishable and frontend API keys for your Clerk production instance to your `.env.production` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard. If you are using [the Sync Host feature](/docs/references/chrome-extension/sync-host), you will also need to set the `PLASMO_PUBLIC_SYNC_HOST` environment variable. The value should be the domain your web app's production server runs on. For example, `https://clerk.com`.
```

Comment:
I opted for `PLASMO_PUBLIC_SYNC_HOST` in the guides to avoid confusion with 'official' env var names that can be read and used by our SDKs. You think it is better to add `CLERK` anyway, to identify its Clerk related?

Reply:
@royanger Makes sense, however, if we consider that the Plasmo build process could be updated to interpolate those env vars, it might be be best to create the standard now and not have to backtrack later.

Also, in talking with some others on the team, from a DX perspective, it makes it very clear, say, in a large env file, that this for Clerk.

Reply:
Sounds good @tmilewski. I'll update these and update the demo repo.